pkg/kn/commands/service/create.go

```diff
@@ -59,7 +59,10 @@ var create_example = `
   kn service create s1 --image dev.local/ns/image:v3 --annotation sidecar.istio.io/inject=false
 
   # Create a private service (that is a service with no external endpoint)
-  kn service create s1 --image dev.local/ns/image:v3 --cluster-local`
+  kn service create s1 --image dev.local/ns/image:v3 --cluster-local
+
+  # Create a service with 250MB memory, 200m CPU requests and a GPU resource limit
+  kn service create gpusvc --image dev.local/ns/image:gpu --requests memory=250Mi,cpu=200m --limits nvidia.com/gpu=1`
```

Comment:
Can we somehow put some link or details on where user can find the correct values to use? talking specifically about things like `nvidia.com/gpu` etc?

Reply:
added a couple of references from upstream in the example